package.json

```diff
@@ -61,28 +61,28 @@
   },
   "dependencies": {
     "@babel/runtime": "7.15.4",
-    "@patternfly/patternfly": "4.132.2",
-    "@patternfly/react-charts": "6.59.2",
-    "@patternfly/react-core": "^4.221.3",
-    "@patternfly/react-icons": "^4.72.3",
-    "@patternfly/react-table": "^4.90.3",
-    "@patternfly/react-tokens": "^4.73.3",
-    "@redhat-cloud-services/frontend-components": "^3.11.6",
-    "@redhat-cloud-services/frontend-components-notifications": "^3.2.11",
-    "@redhat-cloud-services/frontend-components-translations": "^3.2.3",
-    "@redhat-cloud-services/frontend-components-utilities": "3.7.6",
-    "@unleash/proxy-client-react": "^4.1.1",
+    "@patternfly/patternfly": "5.1.0",
+    "@patternfly/react-charts": "7.1.2",
+    "@patternfly/react-core": "^5.1.2",
+    "@patternfly/react-icons": "^5.1.2",
+    "@patternfly/react-table": "^5.1.2",
+    "@patternfly/react-tokens": "^5.1.2",
+    "@redhat-cloud-services/frontend-components": "^4.2.3",
+    "@redhat-cloud-services/frontend-components-notifications": "^4.1.0",
+    "@redhat-cloud-services/frontend-components-translations": "^3.2.7",
+    "@redhat-cloud-services/frontend-components-utilities": "4.0.2",
+    "@unleash/proxy-client-react": "^4.1.2",
     "classnames": "2.3.1",
     "enzyme-to-json": "3.6.2",
```

Comment:
We can safely remove this enzyme

Reply:
good catch, ill remove this before moving forward!